build_modules/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.4.1
```

Comment:
nit: leave `-dev` until we are actually ready to publish (dev dependencies removed) - ideally in the pubspec too

Reply:
Will keep that in mind for next time - I plan to publish right away so I won't bother waiting for another round of travis this time.

Reply:
sgtm